webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -3152,6 +3152,199 @@ export const V1FittingPolicy = {
+    /**
+     * Signed searcher metrics value.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    searcherMetricValue?: number;
+    /**
+     * The id of external run
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalRunId?: number;
+    /**
+     * Trial hyperparameters.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    hyperparameters?: string;
+    /**
+     * summary metrics.
+     * @type {any}
+     * @memberof V1FlatRun
+     */
+    summaryMetrics?: any;
+    /**
+     * The id of the experiment linked to the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    experimentId?: number;
+    /**
+     * The id of the user who created the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    ownerId?: number;
+    /**
+     * Original id of a forked or continued experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    forkedFrom?: number;
+    /**
+     * The id of external experiment
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    externalExperimentId?: number;
+    /**
+     * The resource pool the experiment was created in.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    resourcePool?: string;
+    /**
+     * The current progress of a running experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    experimentProgress?: number;
+    /**
+     * The description of the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    experimentDescription?: string;
+    /**
+     * The experiment name.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    experimentName?: string;
+    /**
+     * The display name of the user that created the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    displayName?: string;
+    /**
+     * The username of the user that created the experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    username?: string;
+    /**
+     * Time in seconds which experiment ran or has been running.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    duration?: number;
+    /**
+     * The id of the project associated with this experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    projectId?: number;
+    /**
+     * The name of the project associated with this experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    projectName?: string;
+    /**
+     * The id of the workspace associated with this experiment.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    workspaceId?: number;
+    /**
+     * The name of the workspace associated with this experiment.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    workspaceName?: string;
+    /**
+     * The archived status of the parent project (can be inherited from workspace).
+     * @type {boolean}
+     * @memberof V1FlatRun
+     */
+    parentArchived?: boolean;
```

Comment:
q: if the run is part of an experiment which is archived inside of a project which is not archived, what would be the value of `parentArchived`?

Reply:
parent archived is true if the parent OR the workpace is archived

Reply:
just want to make sure I understand. a run can have (up to) 3 layers of nested parents:

1. experiment/searcher (optional)
2. project
3. workspace

and `parentArchived` is true if **any** of those are archived?

Reply:
`parentArchived` is only true if 2 & 3 are archived